ui/src/dashboards/components/dashboard_index/DashboardsIndex.tsx

```diff
@@ -229,13 +208,13 @@ class DashboardIndex extends PureComponent<Props, State> {
 
   private get importOverlay(): JSX.Element {
     const {isImportingDashboard} = this.state
+    const {orgs} = this.props
 
     return (
-      <ImportOverlay
-        isVisible={isImportingDashboard}
-        resourceName="Dashboard"
+      <ImportDashboardOverlay
         onDismissOverlay={this.handleToggleImportOverlay}
-        onSubmit={this.handleImportDashboard}
+        orgID={get(orgs, '0.id', '')}
```

Comment:
Why is this defaulting to the first org? Shouldn't we let the user select org?

Reply:
this is defaulting to the first org since that's what creating a dashboard does. In a separate pr, i will add an org dropdown to the import overlay so that a user may choose.

Reply:
#12496